src/internal/storage/server_test.go

```diff
@@ -419,3 +420,28 @@ func checkFilesetCDR(ctx context.Context, t *testing.T, c storage.FilesetClient,
 	}
 	require.Equal(t, 0, len(expected))
 }
+
+func TestRenewFileset(t *testing.T) {
```

Comment:
This seems like it should indicate it is testing the validation. Maybe in the test name, or just a subtest such that we could add other types of subtests later.